README.md

```diff
@@ -1,43 +1,35 @@
-# Proposals related to identity credentials
+# Digital Identities
 
-This repo is being used to work on, and hopefully converge on, a proposal
-related to the invocation of digital identity wallets from the web:
-* [Digital Credential](digital-credentials-2-proposal.md)
+- [Digital identities spec](https://wicg.github.io/digital-identities/)
```

Comment:
```suggestion
- [Digital identities draft spec](https://wicg.github.io/digital-identities/)
```

So that we set the expectation that this is just simply an early draft?